app/controllers/JobController.scala

```diff
@@ -21,7 +21,7 @@ class JobController @Inject()(
 )(implicit ec: ExecutionContext) extends AbstractController(cc) with I18nSupport {
 
   def list = silhouette.SecuredAction.async { implicit request =>
-    jobDAO.findAll(request.identity).map(jobs => Ok(Json.toJson(jobs)))
+    jobDAO.findAll(request.identity).map { jobs => Ok(Json.toJson(jobs)) }
```

Comment:
Not specifically referring to blocks vs parens, but [style guide](https://docs.scala-lang.org/style/declarations.html#spacing) does mention the following under the Function Value section: `Both styles make use of parentheses, since they look clean on a single line.`

Reply:
Do you have preference? I just went with block because more consistent with rest of code and across single / multi lines.

Reply:
mild preference for parens on one-liners unless case match e.g. `tuples.map { case (a,b) => a + b }`, but not really big deal